doc/ref_arch/openstack/chapters/chapter06.md

```diff
@@ -104,10 +104,23 @@ The following rules govern create, read, update, and delete (CRUD) level access.
 ### 6.3.2 System Hardening
 All infrastructure should undergo system hardening, establishes processes to govern the hardening, and documents to cover at a minimal for the following areas:
 
-#### 6.3.2.1 Function
+#### 6.3.2.1 Password policy
+For all infrastructure components, passwords must be hardened and a strict password policy must be applied (req.sec.gen.002).
+
+Passwords must be strengthen:
+-	All vendors default passwords must be changed
+-	Passwords must contain at least 8 characters
+-	Passwords must contain at least one upper case letter, one lower case letter and one non-alphabetic character
+
+For passwords updates, the identity of users must be verified before providing new passwords.
+
+Passwords must be securely stored and transmit in an encrypted format. Password files must be stored separately from application system data.
```

Comment:
```suggestion
Passwords must be encrypted at rest and in-transit. Password files must be stored separately from component and user data.
```

Reply:
@pgoyal01 I agree with the change proposed for the first sentence "Passwords must be encrypted at rest and in-transit.", but I prefer to keep "from application system data"

Reply:
@karinesevilla That is fine